recipe/conda_build_config.yaml

```diff
@@ -172,13 +172,20 @@ docker_image:                                       # [os.environ.get("BUILD_PLA
   - quay.io/condaforge/linux-anvil-aarch64:alma9    # [os.environ.get("CF_CUDA_ENABLED", "False") == "True" and os.environ.get("BUILD_PLATFORM") == "linux-aarch64" and os.environ.get("DEFAULT_LINUX_VERSION", "alma9") == "alma9"]
   - quay.io/condaforge/linux-anvil-ppc64le:alma9    # [os.environ.get("CF_CUDA_ENABLED", "False") == "True" and os.environ.get("BUILD_PLATFORM") == "linux-ppc64le" and os.environ.get("DEFAULT_LINUX_VERSION", "alma9") == "alma9"]
 
+# to be overridden per feedstock; part of zip so that CPU builds can choose different agents than GPU builds
+github_actions_labels:          # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - dummy                       # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - dummy                       # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - dummy                       # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+
 zip_keys:
   -                             # [unix]
     - c_compiler_version        # [unix]
     - cxx_compiler_version      # [unix]
     - fortran_compiler_version  # [unix]
     - cuda_compiler             # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
     - cuda_compiler_version     # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+    - github_actions_labels     # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
```

Comment:
these zip keys have become almost impossible to read.

Reply:
Not sure what alternative you'd have in mind - unless we completely redesign how conda works (or stop supporting CUDA builds), this is irreducible complexity IMO

Reply:
I think we can start with: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6917